msa/black-box-tests/src/test/java/org/thingsboard/server/msa/AbstractContainerTest.java

```diff
@@ -69,13 +69,19 @@ public abstract class AbstractContainerTest {
     protected static final String WSS_URL = "wss://localhost";
     protected static String TB_TOKEN;
     protected static RestClient restClient;
+
+    protected static boolean slowQueue;
+
     protected ObjectMapper mapper = new ObjectMapper();
     protected JsonParser jsonParser = new JsonParser();
 
     @BeforeClass
     public static void before() throws Exception {
         restClient = new RestClient(HTTPS_URL);
         restClient.getRestTemplate().setRequestFactory(getRequestFactoryForSelfSignedCert());
+
+        String QUEUE_TYPE = System.getProperty("blackBoxTests.queue", "kafka");
```

Comment:
why do you need use QUEUE_TYPE as a local variable name? Maybe you intended to use a global variable. If not - just inline this expression and avoid creating a useless variable.